jdaviz/utils.py

```diff
@@ -342,6 +344,85 @@ def indirect_units():
     return units
 
 
+def flux_conversion_general(values, original_unit, target_unit,
+                            equivalencies=None, with_unit=True):
```

Comment:
Should we keep the parameters the same here as in `flux_conversion`? Originally adding `spec` and `slice` had to do with the image-viewers (the one case I explicitly remember testing for is working with the current state here) and eventually image data. I'll need to test this out a bit more to, might not be necessary but I will add a note about this on the Imviz implementation ticket.